generators/server/templates/src/main/java/package/config/OpenApiConfiguration.java.ejs

```diff
@@ -18,7 +18,14 @@ limitations under the License.
 -%>
 package <%= packageName %>.config;
 
+<%
+  let springFox3 = false;
+  if (reactive && (applicationType === 'gateway' || applicationType === 'monolith')) {
+    springFox3 = true;
+  } else { _%>
 import com.google.common.base.Predicates;
+<% } _%>
```

Comment:
I would separate the logic of defining variables and the templating. That's much easier to read and maintain

```suggestion
<%_
  let springFox3 = false;
  if (reactive && (applicationType === 'gateway' || applicationType === 'monolith')) {
    springFox3 = true;
  } 
_%>
<%_ if (!springFox3) { _%>
import com.google.common.base.Predicates;
<% } _%>
```

Reply:
Done.